torchao/experimental/kernels/cpu/aarch64/tests/test_linear.cpp

```diff
@@ -350,4 +354,215 @@ TEST(
   }
 }
 
+// #ifdef TORCHAO_ENABLE_KLEIDI
+// TODO: Wire up the the compile defination for TORCHAO_ENABLE_KLEIDI
+
+template <int weight_nbit, bool has_weight_zeros, bool has_bias, bool has_clamp>
```

Comment:
Is this templating needed for Kleidi?

Reply:
will remove wight_nbit and has_weight_zeros.

has_bias is something we will need. And adding new tests for has_clamp :P